web/book/src/reference/stdlib/other-functions/date.md

```diff
@@ -1,14 +1,11 @@
-# Date and time
-
-We already know that PRQL uses the `@2020-01-01` syntax to declare dates. To
-manipulate those dates, PRQL has a `date` module with some useful functions
+# Date functions
 
 ### `to_text`
 
-This function allows to convert a date into a text. Since there are many
-possible date representations, `to_text` takes a `format` parameter that
-describes thanks to [specifiers](./format-specifiers.md) how the date or
-timestamp should be structured.
+Converts a date into a text.\
```

Comment:
I don't think these are needed in most markdown parsers fwiw

```suggestion
Converts a date into a text.
```

Reply:
It was to enforce the description on one line and more infos under.